packages/e2e-tests/cypress/integration/eligibility/householder-planning/claiming-costs/claiming-costs-steps.js

```diff
@@ -0,0 +1,60 @@
+import {Given,When, Then} from 'cypress-cucumber-preprocessor/steps';
+import {goToAppealsPage} from "../../../../support/common/go-to-page/goToAppealsPage";
+import {verifyPageTitle} from "../../../../support/common/verify-page-title";
+import {verifyPageHeading} from "../../../../support/common/verify-page-heading";
+import {
+  getClaimingCostNo,
+  getClaimingCostsErrorMessage,
+  getClaimingCostYes
+} from "../../../../support/eligibility/page-objects/claiming-costs-po";
+import {getContinueButton} from "../../../../support/householder-planning/appeals-service/page-objects/common-po";
+import {verifyPage} from "../../../../support/common/verifyPage";
+import {verifyErrorMessage} from "../../../../support/common/verify-error-message";
+import {getBackLink, getErrorMessageSummary} from "../../../../support/common-page-objects/common-po";
+const pageUrl = '/before-you-start/claiming-costs-householder';
+const pageTitle = 'Are you claiming costs as part of your appeal? - Before you start - Appeal a planning decision - GOV.UK';
```

Comment:
Householder is missing, though the story is missing this information

```suggestion
const pageTitle = 'Are you claiming costs as part of your appeal? - Before you start - Appeal a householder planning decision - GOV.UK';
```